README.md

```diff
@@ -1,167 +1,141 @@
-* [Prepare a Red Hat Enterprise Linux (RHEL) target with included ISO](#install-red-hat-linux)
-* [Acquire](#acquire-artifacts), [extract, and verify](#validate-artifacts) artifacts within RHEL instance
-* [Install docker](#install-docker) and [load OPAL docker images](#load-docker-images)
-* [Configure](#configure-deployment) and [instantiate OPAL](#instantiate-opal)
+## Prerequisites
 
-## Environment 
+### Environment Requirements
 
-* Open internet connection
-* Python3.9+ with
-  * pip
-  * setuptools >= 43
-* git
+- **Operating System:** Ubuntu (via WSL or standalone Linux) or macOS
+- **Internet Connection:** Required for downloading artifacts
+- **Python:** Version 3.9+
+  - Must include `pip` and `setuptools >= 43`
+- **Git**: Installed and available in the system PATH
+- **Docker & Docker Compose:** Installed and running
 
-Installation and usage is OS-agnostic. We recommend that this package be installed and executed within a [conda](https://conda.io/projects/conda/en/latest/user-guide/install/index.html) environment. 
+> **Note for Windows Users:** If using WSL, ensure you have installed WSL2 and Ubuntu. Install Docker Desktop with WSL integration enabled.
 
-If you are using Windows and python was installed using the [python installer](https://www.python.org/downloads/windows/), the option to add python to the path automatically at install time must have been used. If that option was not used and python was manually added to the path, this script will likely not function as expected. Re-run the installer, choose the modify option, check the box to add python to the user path and complete the installation modification prior to the following steps. 
+## Installation
 
-## Install
+### 1. Setup Python Environment
 
-From within the python environment described in the [Environment](#environment) section, run the command:
+We recommend using a virtual environment:
 
-`pip install git+https://github.com/309thEDDGE/opal-release-downloader.git`
+```sh
+python -m venv opal_env
+source opal_env/bin/activate  # On Windows, use `opal_env\Scripts\activate`
+```
 
-_Note: On windows be sure to use_ `python -m pip install ...`.
+### 2. Install OPAL Release Downloader
 
-This command will pull the source from github, then build and install the python package, including dependencies. It will place several scripts on your path in the active environment. Scripts are described in [Download and verify artifacts](#download-and-validate-artifacts).
+```sh
+pip install git+https://github.com/309thEDDGE/opal-release-downloader.git
+```
 
-## Usage
+## Download and Verify OPAL Artifacts
 
-### Runtime arguments: 
+1. **Navigate to an empty directory** where OPAL artifacts will be stored.
+2. **Ensure you are in the correct Python environment.**
+3. **View script options:**
+   ```sh
+   download_opal_artifacts -h
+   ```
+4. **Download artifacts:**
+   ```sh
+   download_opal_artifacts opal-deployment-artifacts
```

Comment:
For now, lets not give the bucket_name "opal-deployment-artifacts" in the documentation, for security reasons. Let's say they need to get it from OPAL developers